templates/scripts/dl-binary.sh.j2

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# vim: ft=bash
+set -e
+
+GH_NIMBUS_RELEASE_URL="https://api.github.com/repos/status-im/nimbus-eth2/releases/latest"
+GH_NIMBUS_DOWNLOAD_URL="https://github.com/status-im/nimbus-eth2/releases/latest/download/"
+ARCH="amd64"
+BEACON_NODE_SERVICE="{{ beacon_node_service_name }}.service"
+BUILD_SERVICE="build-{{ beacon_node_service_name }}.service"
```

Comment:
Shall we rename the service too?

Reply:
I think we should have a cleanup of old `build-` services, timers, scripts and consul entries in the role, at least until we clean it all up.

Reply:
> Shall we rename the service to `update-` too?

This variable is used only to check if no service `buid-` is running.
It was to avoid issue if the host wasn't clear. It shouldn't be necessary now since i change the binary location to another folder than `repo/build`.

Reply:
> I think we should have a cleanup of old `build-` services, timers, scripts and consul entries in the role, at least until we clean it all up.

I listed in the issue the command to clean the host from the `build-` service. I can add a role to clean it automaticly